oe_whitelabel.info.yml

```diff
@@ -15,7 +15,7 @@ libraries-override:
         css/system.admin.css: false
 
 ckeditor_stylesheets:
-  - assets/css/oe-bcl-default.min.css
+  - assets/css/oe_whitelabel.style.min.css
```

Comment:
We don't define any css in whitelabel that could impact the wysiwyg, we can drop this line.